lib/Command/ConfigReport.php

```diff
@@ -30,8 +30,25 @@
  */
 class ConfigReport extends Command {
 
-	public function __construct() {
+	/**
+	 * @var ReportDataCollector
+	 */
+	private $reportDataCollector;
 
+	public function __construct() {
```

Comment:
Why would it generate a full report on instantiation? o.O

It seems like the commands are instantiated before `Application` had the chance to register its own services. Therefore alias does not work here either.